plan.go

```diff
@@ -42,6 +42,10 @@ type Plan struct {
 	// this plan.
 	PlannedValues *StateValues `json:"planned_values,omitempty"`
 
+    // The change operations for resources and data sources within this plan
+    // resulting from resource drift.
+    ResourceDrift []*ResourceChange `json:"resource_drift,omitempty"`
+
```

Comment:
```suggestion
	// The change operations for resources and data sources within this plan
	// resulting from resource drift.
	ResourceDrift []*ResourceChange `json:"resource_drift,omitempty"`

```

`go fmt ./...` in the IDE might help.

Reply:
I have `go fmt` running on save in my editor. Weird it didn't properly format